test/helper.lua

```diff
@@ -166,7 +174,11 @@ function helpers.create_space_with_hash_index(engine)
 end
 
 function helpers.create_space_with_bitset_index(engine)
-    local space = create_space("bitset", engine)
+    return helpers.create_named_space_with_bitset_index("bitset", engine)
+end
+
+function helpers.create_named_space_with_bitset_index(name, engine)
+    local space = create_space(name, engine)
     space:create_index("primary", {
         type = "TREE",
```

Comment:
Please explain why do we need these changes.

Reply:
This is necessary to move data(archive) from the expiration space to the archive, as in the original. Of course, we can somehow reuse the space and insert data, for example, without an expiration time into the same space, but this is at your discretion

https://github.com/ArtDu/expirationd/blob/bcba74ca00f593bd6922e66b81a7125c228ba983/test/unit/expiration_process_test.lua#L43-L51

Reply:
> This is necessary to move data(archive) from the expiration space to the archive, as in the original. Of course, we can somehow reuse the space and insert data, for example, without an expiration time into the same space, but this is at your discretion

Still don't get it why we should change helpers that used in test setup in a whole test suite.

Reply:
We can create a new space right in the test if you want, I just thought you can use the code so that there is no duplicate code. And now, through helpers, I cannot create 2 spaces with different names